sambacc/commands/run.py

```diff
@@ -67,6 +108,12 @@ def run_container(ctx: Context) -> None:
     """Run a specified server process."""
     if ctx.cli.no_init and ctx.cli.setup:
         raise Fail("can not specify both --no-init and --setup")
+
+    if ctx.cli.wait_for:
+        conditions = [_wait_for_conditions[n]() for n in ctx.cli.wait_for]
+        while not all(c.met(ctx) for c in conditions):
+            time.sleep(1)
+
```

Comment:
This has the potential of forever loop. Better limit the number of iterations/time.

Reply:
That was intentional. But I suppose having a very long timeout is ok because something like 5 minutes would clearly be outside the expected bounds of startup and provide some diagnostic information in the failure.
I'll make this change.

Reply:
done. ptal.